### ISSUE TITLE
Background color of selected text in Info inspector

### ISSUE THREAD

**Issue by LennyLip** [losnoco/Cog#321]
**Describe**
When I select text in Info Inspector it's almost not visible due background color (when Info Inspector is active window)

<img width="250" alt="Screenshot 2022-08-10 at 07 42 26" src="https://user-images.githubusercontent.com/955994/183817865-222e37f2-7900-4a63-8bb3-1ba236c6d26c.png">

**Comment by kode54** [comment]
The Info Inspector is supposed to be forced dark style, and the selection color is supposed to be a dark color based on your accent color, or the app's default accent color if the system is configured to allow it.

**Comment by LennyLip** [comment]
I have default settings about colours, contrast, etc
<img width="421" alt="Screenshot 2022-08-10 at 09 42 38" src="https://user-images.githubusercontent.com/955994/183833164-2f41fc48-00f6-4ed4-83b3-6535b64c2446.png">

**Comment by LennyLip** [comment]
> Edit 3: Here it is with Increase Contrast turned off: 

Your black background is not common, look at the picture in the repository. It looks exactly like my screenshot. https://github.com/losnoco/Cog/blob/main/.github/images/MainWindow.png

**Comment by kode54** [comment]
I'm done.

**Comment by LennyLip** [comment]
> I'm done.
Ok, thank you for your hard work.

**Comment by kode54** [comment]
Yes, and with a white background, it should use that dark orange as the selection background color. Please try this again. If it isn't fixed, it's an Old macOS bug that I can't adequately deal with.

**Comment by LennyLip** [comment]
<img width="376" alt="Screenshot 2023-10-12 at 18 27 30" src="https://github.com/losnoco/Cog/assets/955994/f0fe4961-f69f-49f7-a8ad-5ee1773ff6af">
Same here, macOS 13.5.2 (22G91)

**Comment by kode54** [comment]
Guess I have to determine why the colors are overriding the settings. Either I made it forced dark mode for the info box, in which case the white text makes sense, but the selection color does not. Or it's not forced dark mode, and the text is accidentally set to forced white color instead of theme default.

**Comment by kode54** [comment]
Cool, I've verified that the color being unreadably bright only affects Intel Macs, as it happens on my 2014 MacBook Pro, but not on my 2020 Apple Silicon Mac mini.

**Comment by LennyLip** [comment]
> Cool, I've verified that the color being unreadably bright only affects Intel Macs, as it happens on my 2014 MacBook Pro, but not on my 2020 Apple Silicon Mac mini.

I have a MacBook air with M1 chip.

**Comment by kode54** [comment]
In that case, maybe it's specific to laptops. I can't say it's specific to P3 screens, since I don't think the 2014 MBP has a P3 screen.

**Comment by kode54** [comment]
I'll attempt to make the Info Inspector light/dark switching next time I'm working on the Mac.

**Comment by LennyLip** [comment]
> In that case, maybe it's specific to laptops. I can't say it's specific to P3 screens, since I don't think the 2014 MBP has a P3 screen.

I have an external HUAWEI MateView monitor, and it looks exactly as at laptop.

**Comment by kode54** [comment]
I guess I didn't test light mode recently. It was working before. Will deal with it soon.
